src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement, Ref } from 'react';
+import { FiSkipBack, FiSkipForward } from 'react-icons/fi';
+import ReactSlider from 'react-slider';
+import { Domain, extendDomain, ScaleType } from '../../../../packages/lib';
+import { createAxisScale } from '../../../vis-packs/core/utils';
+import styles from './DomainSlider.module.css';
+import Thumb from './Thumb';
+
+const SLIDER_RANGE: Domain = [1, 100];
+
+interface Props {
+  rawDomain: Domain;
+  dataDomain: Domain;
+  disabled?: boolean;
+  isAutoMin: boolean;
+  isAutoMax: boolean;
+  onChange: (domain: Domain) => void;
+  onAfterChange: (domain: Domain) => void;
+}
+
+const EXTEND_FACTOR = 0.2;
+
+function ScaledSlider(props: Props): ReactElement {
+  const { rawDomain, dataDomain, disabled, isAutoMin, isAutoMax } = props;
```

Comment:
`rawDomain` could be `unscaledValue` or even `value` as we don't need to know that a scale is used then ? :shrug:

Reply:
It was a bit too confusing when I had `useCallback`s, but now it's fine, you're right 👍